tutorial/source/conf.py

```diff
@@ -68,9 +71,10 @@
 # built documents.
 #
 # The short X.Y version.
-version = u''
-# The full version, including alpha/beta/rc tags.
-release = u''
+# get the latest git tag, otherwise get the latest commit
+version = re.sub('^v', '', os.popen('git describe --tags').read().strip())
```

Comment:
I usually use `check_call('git describe --tags'.split()).strip()` from `subprocess` module. Is there any difference between this and `os.popen()`.

Reply:
Sorry, could you add the comment with expected output below this line?

Reply:
> When I run it I get pytorch-0.3-28-g4c52460 and I'm not sure whether this is expected

it is. basically the last tag and the commit in which it corresponds to (for those bleeding-edge users)

> Is there any difference between this and os.popen()

not anything significant.. i can change it if you have a reason that incantation is better

>  could you add the comment

done